appveyor.yml

```diff
@@ -0,0 +1,52 @@
+#
+# Support PHP versions: http://windows.php.net/downloads/releases/
+#
+build: false
+clone_depth: 1
+clone_folder: C:\projects\php-coveralls
+
+platform:
+  # x86
+  - x64
+
+environment:
+  matrix:
+    # php_ver: 5.5.38
+    - php_ver: 5.6.30
+    # php_ver: 7.0.16
+    - php_ver: 7.1.3
+
+cache:
+  - '%APPDATA%\Composer'
+  - C:\ProgramData\chocolatey\bin -> appveyor.yml
+  - C:\ProgramData\chocolatey\lib -> appveyor.yml
+  - C:\projects\php-coveralls\vendor
+  - C:\Tools\composer.phar
+  - C:\Tools\php -> appveyor.yml
```

Comment:
I'm sorry for my poor English. 
platform, environment-matrix and cache set chache and test cases.